### ISSUE TITLE
Applet is glitchy on Ubuntu MATE status bar.

### ISSUE THREAD

**Issue by djvar94** [sindresorhus/caprine#1301]
When I click on the applet for Caprine on the Statusbar it opens a blank option menu.
Can't really take a screenshot of that sorry but I hope you get what I mean..

**Comment by wereii** [comment]
I am too using MATE and it works nicely for me, both snap one and self-build one.
What distro and version are you on?

My is (Ubuntu MATE)
```
~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 19.10
Release:	19.10
Codename:	eoan
```

**Comment by djvar94** [comment]
@wereii 
LSB Version:	core-9.20170808ubuntu1-noarch:printing-9.20170808ubuntu1-noarch:security-9.20170808ubuntu1-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.4 LTS
Release:	18.04
Codename:	bionic

Maybe it's a MATE bug, maybe as you're using 19.10 you should be having a newer version of MATE Desktop. So maybe by 20.04 it will be fixed for me in a few weeks...

**Comment by wereii** [comment]
`apt show ubuntu-mate-desktop` says `Version: 1.257`

I've had similar problem with Steam, after clicking on the applet, blank menu with 0 items would pop up. I've resolved it following [this issue](https://github.com/ValveSoftware/steam-for-linux/issues/4797). Only some packages needed to be installed, maybe some session restart (I think I've simply rebooted the computer to be sure).

Yeah, this is Caprine, not Steam but if my knowledge is right, there is some similarity as both Steam and Electron use Chromium Embedded Framework.

So you might as well try it.
